tests/regtest_spec.py

```diff
@@ -1,92 +1,111 @@
 #!/usr/bin/python
 """Print a test spec on stdout.
```

Comment:
Added five lines to regtest_spec.py with the demo- prefix. "./demo.sh run" will only run those, sequentially, with output directed to the console. It works fine except the timing information is not reported (as it is not logged).